src/core/datagram.c

```diff
@@ -372,7 +372,7 @@ QuicDatagramQueueSend(
                 AllocFailure,
                 "Allocation of '%s' failed. (%llu bytes)",
                 "DATAGRAM_SEND operation",
-                0);
+                UINT64_C(0));
```

Comment:
Perhaps instead of all these, we use `0ULL`? That works, right?

Reply:
Yes, that would work. Will make the change.